tests/credentialDefinition.test.spec.ts

```diff
@@ -49,7 +49,7 @@ describe('CredentialDefinitionController', () => {
       expect(response.statusCode).to.be.equal(400)
     })
 
-    test('should return 404 NotFound when credential definition not found', async () => {
+    test.skip('should return 404 NotFound when credential definition not found', async () => {
```

Comment:
why is this skipped?

Reply:
You told me to skip it ;) Removing .skip will produce errors due to the fact that the test tries to connect to a real chain that has some issues..